rest-openehr/src/main/java/org/ehrbase/rest/openehr/OpenehrDefinitionQueryController.java

```diff
@@ -107,11 +115,10 @@ public ResponseEntity<QueryDefinitionResponseData> getStoredQueryVersion(
     }
 
     @EhrbaseAuthorization(permission = EhrbasePermission.EHRBASE_QUERY_CREATE)
-    @RequestMapping(
-            value = {"/{qualified_query_name}/{version}", "/{qualified_query_name}"},
-            method = RequestMethod.PUT)
+    @PutMapping(value = {"/{qualified_query_name}/{version}", "/{qualified_query_name}"})
     @Override
     public ResponseEntity<QueryDefinitionResponseData> putStoreQuery(
+            @RequestHeader(value = CONTENT_TYPE, required = false) String contentType,
             @RequestHeader(value = ACCEPT, required = false) String accept,
             @PathVariable(value = "qualified_query_name") String qualifiedQueryName,
             @PathVariable(value = "version") Optional<String> version,
```

Comment:
The version needs to follow the SEMVER syntax, but we never check it.
This is also true for the client.
This could be also taken care of in another ticket.